dev/ci/ci-unimath.sh

```diff
@@ -1,14 +1,13 @@
 #!/usr/bin/env bash
 
-ci_dir="$(dirname "$0")"
-source ${ci_dir}/ci-common.sh
+ci_dir=$(dirname "$0") &&
+. "${ci_dir}"/ci-common.sh &&
```

Comment:
Please keep `source`.

Reply:
Source is less portable than ., so no.

Reply:
Depending on set -e for the semantics of your program leads to unreliable scripts, because you lose the capability to copy-paste commands from any place and retain the same semantics. 

I am not going to argue with you, since I am *way* more experienced than you are.

Reply:
Well, I regret not having Coq's CoC ready but certainly I don't find this way of addressing a core developer is not appropriate.

I am not going to argue with you since I am _way_ more experienced with Coq than you are ;)

Reply:
Coq is just another program. Clearly, whoever wrote this script was an amateur.

Does Coq's CoC state that worse programs are better, just because they have been implemented by someone who is more politically connected?